app/workers/education_form/generate_spool_files.rb

```diff
@@ -18,6 +19,29 @@ def delete_malformed_claims
       { count: count, filename: filename }
     end
 
+    def full_name(name)
+      return '' if name.nil?
+      [name['first'], name['middle'], name['last'], name['suffix']].compact.join(' ')
+    end
+
+    def get_names_and_ssns
+      names_and_ssns = []
+
+      valid_records.find_each do |education_benefits_claim|
+        next if education_benefits_claim.region == :eastern
```

Comment:
I don't think it would hurt to include these. We can tell them that the Eastern region ones are processed already, but they can use them to check if they'd like.